test/dotnet-new3.UnitTests/dotnet-new3.IntegrationTests.csproj

```diff
@@ -11,6 +11,39 @@
     <PackageReference Include="Wcwidth.Sources" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Common.ProjectTemplates.2.1" Version="5.0" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.DotNet.Common.ProjectTemplates.3.1" Version="5.0" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.DotNet.Common.ProjectTemplates.5.0" Version="5.0" GeneratePathProperty="true" />
```

Comment:
Maybe also pull 6.0 prerelease package and include in the tests?

Reply:
Same as above...